docs/Plugins.rst

```diff
@@ -1998,11 +1998,11 @@ Regrows all the grass. Not much to it ;)
 restrictice
 ===========
 Restrict traffic on all tiles on top of visible ice.
-See also `alltraffic`, `filltraffic`, and `restrictliquid`.
+See also `alltraffic`, `filltraffic`, and `restrictliquids`.
 
-.. _restrictliquid:
+.. _restrictliquids:
 
-restrictliquid
+restrictliquids
```

Comment:
It does - the underline must be at least as long as the title, and *may* be longer (but that looks messy).